deployment/ansible/roles/ckan-odp-configuration/templates/backup.sh.j2

```diff
@@ -14,5 +17,6 @@ pushd $CKAN_DIR
 
 $PASTER db dump --config=$CKAN_CONFIG $BACKUP_FILE
 gzip -f -9 $BACKUP_FILE
+s3cmd -c $S3CMD_CFG put $BACKUP_FILE.gz s3://$S3_BUCKET/$DATE-$HOSTNAME.gz
 
 popd
```

Comment:
If `s3cmd` doesn't auto-delete after pushing things to S3, then we should probably remove the backup file as the last step in this script, otherwise we risk filling up the disk with backups.